docs/development/i18n.md

```diff
@@ -2,17 +2,17 @@
 
 !!! tldr "Django docs"
 
-    [Internationalization and localization](https://docs.djangoproject.com/en/4.0/topics/i18n/)
+    [Internationalization and localization](https://docs.djangoproject.com/en/5.0/topics/i18n/)
 
-    [Translation](https://docs.djangoproject.com/en/4.0/topics/i18n/translation/)
+    [Translation](https://docs.djangoproject.com/en/5.0/topics/i18n/translation/)
 
 !!! example "Message files"
 
     English messages: [benefits/locale/en/LC_MESSAGES/django.po](https://github.com/cal-itp/benefits/blob/dev/benefits/locale/en/LC_MESSAGES/django.po)
 
 The Cal-ITP Benefits application is fully internationalized and available in both English and Spanish.
 
-It uses Django's built-in support for translation using [message files](https://docs.djangoproject.com/en/4.0/topics/i18n/#term-message-file), which contain entries of `msgid`/`msgstr` pairs. The `msgid` is referenced in source code so that Django takes care of showing the `msgstr` for the user's language.
+It uses Django's built-in support for translation using [message files](https://docs.djangoproject.com/en/5S.0/topics/i18n/#term-message-file), which contain entries of `msgid`/`msgstr` pairs. The `msgid` is referenced in source code so that Django takes care of showing the `msgstr` for the user's language.
```

Comment:
```suggestion
It uses Django's built-in support for translation using [message files](https://docs.djangoproject.com/en/5.0/topics/i18n/#term-message-file), which contain entries of `msgid`/`msgstr` pairs. The `msgid` is referenced in source code so that Django takes care of showing the `msgstr` for the user's language.
```

Reply:
Oops! Must have been a fat-finger find/replace :sweat_smile: